nais/dev-fss.yml

```diff
@@ -26,7 +26,8 @@ spec:
       cpu: 200m
       memory: 512Mi
   ingresses:
-    - https://pleiepenger-dokument.nais.preprod.local
+    - https://pleiepenger-dokument.nais.preprod.local #TODO: Rename ingress
+    - https://k9-dokument.nais.preprod.local
```

Comment:
Dette er den eneste nais-filen med to ingresser. Må endres i de tre andre på samme måte også, at pleiepenger-dokument & k9-dokument ingressene ligger samtidig.